unlock-app/src/components/interface/checkout/main/Payment.tsx

```diff
@@ -63,17 +64,35 @@ export function Payment({ injectedProvider, checkoutService }: Props) {
     }
   )
 
-  const { isLoading: isClaimableLoading, isClaimable } = useIsClaimable({
-    lockAddress: lock.address,
-    network: lock.network,
-  })
-
   const { isLoading: isBalanceLoading, data: balance } = useBalance({
     account: account!,
     network: lock.network,
     currencyContractAddress: lock.currencyContractAddress,
   })
 
+  const { data: purchaseData, isLoading: isPurchaseDataLoading } =
+    usePurchaseData({
```

Comment:
I assume this is cached right? we seem to do that in many many places :)

Reply:
Yes